test/forkMainnet/AMMWrapper/Signing.t.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "test/utils/StrategySharedSetup.sol";
+
+function computeEIP712DomainSeparator(address verifyingContract) returns (bytes32) {
+    uint256 CHAIN_ID = 1;
```

Comment:
Chain ID is also fixed so Foundry or sdk don't have to coordinate on which network to test against. It's just pure signing, network environment is irrelevant.